datreeio/datree
# build: bump `gopsutil` package version to 3.22 version

Fixes: #512

**Changes Proposed in the PR**
1. Updated the `gopsutils` package version to 3.22 to fix deprecation warning.

**Testing Instruction**
1. Build project and verify during build we are not getting any warning.
2. Test existing functionality of the tool i.e. it is working as expected.
